tests/unit/test_auth.py

```diff
@@ -55,6 +54,7 @@ def test_u2m(self):
         assert headers == headers2
 
 
+@pytest.mark.skip(reason="Broken after rewriting auth")
```

Comment:
Should I address these or is it fine to be broken?

Reply:
We want to test token auth, but we no longer use ._provider_from_dict, so we need a new test.